src/Apps/gNtpConv.cxx

```diff
@@ -398,6 +398,11 @@ void ConvertToGST(void)
                                  //  - ((e/h) * energy)   for pi0, gamma, e-, e+, where e/h is set to 1.3
                                  //  - (kinetic energy) for other particles
 
+  double brXSec;              // the event cross section in 1E-38cm^2
```

Comment:
Following https://root.cern.ch/doc/master/classTTree.html I would recommend to define these variables as Double_t